pom.xml

```diff
@@ -398,12 +401,27 @@
         <version>${jettison.version}</version>
       </dependency>
 
+      <dependency>
+        <groupId>org.apache.hadoop</groupId>
+        <artifactId>${hadoop-client-api.artifact}</artifactId>
+        <version>${hadoop.version}</version>
+        <scope>${hadoop.deps.scope}</scope>
+      </dependency>
+
       <dependency>
         <groupId>org.apache.hadoop</groupId>
         <artifactId>hadoop-client</artifactId>
```

Comment:
Is this dependency still needed ? It will be a duplicate of the newly added above (line 404-409)

Reply:
Fixed